consensus/propagation/types/unmarshalling_test.go

```diff
@@ -173,10 +266,192 @@ func FuzzTxsToParts(f *testing.F) {
 			}
 		}
 
-		parts := TxsToParts(subset)
+		lastPart := partSet.GetPart(int(partSet.Total() - 1))
+		parts, err := TxsToParts(subset, partSet.Total(), types.BlockPartSizeBytes, uint32(len(lastPart.Bytes)))
+		require.NoError(t, err)
 		for _, part := range parts {
 			expectedPart := partSet.GetPart(int(part.Index))
 			require.Equal(t, expectedPart.Bytes, part.Bytes)
 		}
 	})
 }
+
+func TestTxsToParts_Panic(t *testing.T) {
```

Comment:
silly question but it looks like this test doesn't assert that a `panic` occurs so why the `_Panic` suffix?

Reply:
nvm, maybe this test is asserting that no panic is encountered